### PR TITLE
Refactoring: make game loop and scenes consistent

### PR DESCRIPTION
**Pull request by gsomix**

### CODE REVIEW

File: O21.Game/Scenes/PlayScene.fs [hunk 1]
```diff
@@ -1,44 +1,52 @@
...
+open System.Numerics
 open Raylib_CsLo
 open type Raylib_CsLo.Raylib
 
 open O21.Game
 open O21.Game.U95
 open O21.Game.U95.Parser
 
-type PlayScene() =
+type PlayScene = {
```
Comment (sotiredofyours) [inline O21.Game/Scenes/PlayScene.fs:11]: I think, we can add HUD to PlayScene`s labels

**Review by ForNeVeR — APPROVED** [review]
What I see looks good.

**Comment by ForNeVeR** [comment]
@gsomix, will you rebase/merge this with `main`, or do you need help? I can do it, no worries.

**Comment by gsomix** [comment]
@ForNeVeR Yes, I'm going to rebase it by myself.

**Review by ForNeVeR — APPROVED** [review]
Thanks!

**Pull request merged**